example/position.py

```diff
@@ -18,7 +18,7 @@ def main(stdscr):
         position=pick.Position(y=y, x=0)  # comment this to demonstrate the issue it solves
     )
 
-    stdscr.addstr(f"\nYou choosed {option} at index {index}\n")
+    stdscr.addstr(f"\nYou chosen {option} at index {index}\n")
```

Comment:
I think it should be chose:

```suggestion
    stdscr.addstr(f"\nYou chose {option} at index {index}\n")
```